packages/pwa-kit-react-sdk/src/utils/opentelemetry.js

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {trace, context, SpanStatusCode} from '@opentelemetry/api'
+import {hrTimeToMilliseconds, hrTimeToTimeStamp} from '@opentelemetry/core'
+import logger from './logger-instance'
+
+const DEFAULT_SERVICE_NAME = 'pwa-kit-react-sdk'
+
+export const OTEL_CONFIG = {
+    serviceName: process.env.OTEL_SERVICE_NAME || DEFAULT_SERVICE_NAME,
+    enabled: process.env.OTEL_SDK_ENABLED === 'true',
+    b3TracingEnabled: process.env.OTEL_B3_TRACING_ENABLED === 'true'
+}
+
+export const getServiceName = () => OTEL_CONFIG.serviceName
+
+const SERVICE_NAME = OTEL_CONFIG.serviceName
```

Comment:
```suggestion
```

We don't need this line. As Adam mentioned, we can replace all instances of `SERVICE_NAME` with `getServiceName()`